src/sentry/static/sentry/app/views/dashboardsV2/detail.tsx

```diff
@@ -247,6 +247,14 @@ class DashboardDetail extends React.Component<Props, State> {
     return (
       <GlobalSelectionHeader
         skipLoadLastUsed={organization.features.includes('global-views')}
+        defaultSelection={{
+          datetime: {
+            start: null,
+            end: null,
+            utc: false,
+            period: DEFAULT_STATS_PERIOD,
```

Comment:
Just a note here but make sure you're not using `period` from the URL directly if you do this. The global selection header will not add the period into the URL parameters if it isn't there already. This means that the Reflux store and the URL parameters will be out of sync.

Reply:
Good point! I did a check of the dashboard widgets and they are all reading from `selection.period`